src/s2/util/math/vector.h

```diff
@@ -175,8 +175,7 @@ class BasicVector {
 
   // Euclidean norm. For integer T, correct only if Norm2 does not overflow.
   FloatType Norm() const {
-    using std::sqrt;
-    return sqrt(Norm2());
+    return std::sqrt(Norm2());
```

Comment:
Are these changes to fix errors, warnings, or style?

Reply:
Warnings about mismatched attributes. There are still some others elsewhere in the codebase (and also in the Abseil code), but they did not have the same simple fix.